app/models/patient.rb

```diff
@@ -131,6 +131,10 @@ def relationship_to(parent:)
     parent_relationships.find { _1.parent == parent }
   end
 
+  def initials
```

Comment:
Doesn't really matter, but might be nice to put this in the `FullNameConcern` since that relates to formatting names (and although unlikely to be used allows this method to exist for consent forms and users as well).

Reply:
done